projects/storefrontlib/src/cms-components/product/product-tabs/product-attributes/product-attributes.component.spec.ts

```diff
@@ -1,18 +1,59 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-import { I18nTestingModule, Product } from '@spartacus/core';
+import {
+  I18nTestingModule,
+  Product,
+  Feature,
+  FeatureUnit,
+  Classification,
+  FeatureValue,
+} from '@spartacus/core';
 import { Observable, of } from 'rxjs';
 import { CurrentProductService } from '../../current-product.service';
 import { ProductAttributesComponent } from './product-attributes.component';
+import { By } from '@angular/platform-browser';
 
-const mockProduct: Product = { name: 'mockProduct' };
+const createMock = function (mockFeatures: Feature[]): Product {
+  let mockClass: Classification = { features: mockFeatures };
```

Comment:
For ts-lint use 'const' instead of 'let' for `mockClass` and `mockProduct` since they aren't reassigned.